draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -211,16 +211,17 @@ single DCCP connection with no inherent performance penalty
 for application subflows that do not require in-ordered delivery. DCCP
 does not provide built-in support for those multiple application subflows.
 
-In the following, the term "subflow" refers to physical
-separate DCCP subflows transmitted via different paths, but not to
-application subflows. Application subflows are differing content-wise
+In the following, use of the term subflow will refer to DCCP subflows transmitted 
```

Comment:
```suggestion
In the following, the term subflow refers to DCCP subflows transmitted 
```

Reply:
also here: thanks, fine - I think we can commit the suggested changes and resolve the issue, right? also @markusa @abrunstrom